lib/home-manager-configuration.nix

```diff
@@ -0,0 +1,21 @@
+{ self, lib, home-manager }:
+
+{ configuration
+, system
+, homeDirectory
+, username
```

Comment:
Some of those aren't used below.

Reply:
Yes but they are required by home-manager so I decided to expose them on our end so we can document them.